npm/pkg/dataplane/policies/policy_windows.go

```diff
@@ -148,24 +145,18 @@ func getAddrListFromSetInfo(setInfoList []SetInfo) string {
 	return setInfoStr
 }
 
-func getPortStrFromPorts(ports []Ports) string {
+func getPortStrFromPorts(port Ports) string {
 	portStr := ""
-	for i, port := range ports {
-		if port.Port == 0 {
-			continue
-		}
-		tempPortStr := fmt.Sprintf("%d", port.Port)
-		if port.EndPort != 0 {
-			for tempPort := port.Port + 1; tempPort <= port.EndPort; tempPort++ {
-				tempPortStr += fmt.Sprintf(",%d", tempPort)
-			}
-		}
-		if i < len(ports)-1 {
-			portStr += tempPortStr + ","
-		} else {
-			portStr += tempPortStr
+	if port.Port == 0 {
+		return portStr
+	}
+	tempPortStr := fmt.Sprintf("%d", port.Port)
```

Comment:
Seems like keep using`portStr` is better without `tempPortStr`.
Btw, does windows need to list up all port numbers unlike linux can use `<port>:<EndPort>"